cmd/bee/cmd/start_windows.go

```diff
@@ -2,6 +2,7 @@
 // Use of this source code is governed by a BSD-style
 // license that can be found in the LICENSE file.
 
+//go:build windows
```

Comment:
is this needed?

Reply:
It seems that starting go1.18 the syntax for build constraints has changed https://pkg.go.dev/cmd/go#hdr-Build_constraints. It's likely that the old syntax will be phased out in the future so we need to get on-board at some point, might as well be now. @vandot  wdyt?

Reply:
```
In Go 1.18, go fix [go tool fix] now removes the now-obsolete // +build lines in modules declaring go 1.18 or later in their go.mod files.
```

Reply:
As @notanatol said, it started in go1.17 and probably they will deprecate old syntax at some point